commit cc09c5bcf130f7c6e17b824a06cbc9f2c1fbb060
Author: Jason Gaston <jason.d.gaston@intel.com>
Date:   Mon Jan 28 17:24:37 2008 -0800

    PCI: irq: patch for Intel ICH10 DeviceID's
    
    This patch adds the Intel ICH10 LPC Controller DeviceID's.
    
    Signed-off-by: Jason Gaston <jason.d.gaston@intel.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

Adjusted to apply to Debian's 2.6.24 by dann frazier <dannf@hp.com>

diff -urpN linux-source-2.6.24.orig/arch/x86/pci/irq.c linux-source-2.6.24/arch/x86/pci/irq.c
--- linux-source-2.6.24.orig/arch/x86/pci/irq.c	2008-01-24 15:58:37.000000000 -0700
+++ linux-source-2.6.24/arch/x86/pci/irq.c	2008-05-01 17:44:12.000000000 -0600
@@ -571,6 +571,10 @@ static __init int intel_router_probe(str
 		case PCI_DEVICE_ID_INTEL_ICH9_4:
 		case PCI_DEVICE_ID_INTEL_ICH9_5:
 		case PCI_DEVICE_ID_INTEL_TOLAPAI_0:
+		case PCI_DEVICE_ID_INTEL_ICH10_0:
+		case PCI_DEVICE_ID_INTEL_ICH10_1:
+		case PCI_DEVICE_ID_INTEL_ICH10_2:
+		case PCI_DEVICE_ID_INTEL_ICH10_3:
 			r->name = "PIIX/ICH";
 			r->get = pirq_piix_get;
 			r->set = pirq_piix_set;
